control-plane/csi-driver/src/bin/node/nodeplugin_nvme.rs

```diff
@@ -36,6 +67,14 @@ impl NvmeOperations for NvmeOperationsSvc {
             .await
             .map_err(|error| warn!(error=%error, "Failed to do fixup after connect"));
 
+        {
```

Comment:
hmm, I don't think this logic is sound. If the future is cancelled after getting the entry but before this next block of code we end up "leaking" the entry?
To fix this we could use the Drop trait instead.

Reply:
Yeah, I was also unsure of this bit working, but on which structure should I implement the drop trait?

Reply:
Have implemented as discussed.